CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Changed
 
+- MNT: Refactor flight prints module [#758](https://github.com/RocketPy-Team/RocketPy/pull/578)
```

Comment:
should be 578 mot 758

Reply:
[ad189a8](https://github.com/RocketPy-Team/RocketPy/pull/579/commits/ad189a8a16c4e7f7f2039105e1e6630d226ed12e)